cocos/renderer/pipeline/forward/ForwardPipeline.h

```diff
@@ -62,6 +62,7 @@ class CC_DLL ForwardPipeline : public RenderPipeline {
     void setSkybox(uint);
     void setShadows(uint);
     void destroyShadowFrameBuffers();
+    uint8_t getCombinationSignY();
```

Comment:
getCombinationSignY -> getCombinationSignY const.
And `combinatonSignY` seems confusion.

Reply:
confusion? Do you have a better suggestion? I will add an explanation to this function

Reply:
It is not just a sign to indicate direction. It is more about to convert value from one region to another region. But i do not have better suggestion. May be can ask @YunHsiao or @ Jacksparrow2525 's help.

Reply:
It is not just a sign to indicate direction. It is more about to convert value from one region to another region. But i do not have better suggestion. May be can ask @YunHsiao or @ Jacksparrow2525 's help.

Reply:
I think combinatonSignY is okay, because it will be parsed on the shader side

Reply:
k, please add description for it. And add const.

Reply:
And i think it only needs to be computed once, not every frame.

Reply:
Yes, I agree, so I will add a private variable for assignment